api/types/types.go

```diff
@@ -38,17 +40,18 @@ type ImageInspect struct {
 	DockerVersion   string
 	Author          string
 	Config          *container.Config
-	Architecture    string
-	Variant         string `json:",omitempty"`
-	Os              string
-	OsVersion       string `json:",omitempty"`
-	Size            int64
```

Comment:
Need to think of this one, as this will have different output in JSON (`Architecture => architecture`, `OsVersion -> os.version`, `Variant -> variant`).

The "capitalised" vs "lowercase" changes should not be a problem when (un)marshalling, as Golang handles these case-insensitive, but `OsVersion -> os.version` _will_ (likely) be an issue.